result-dev

```diff
@@ -0,0 +1 @@
+/nix/store/gf08zxy0445dvjs3cw2hi3r8lavfbfiq-libgit2-1.8.1-dev
```

Comment:
question: is this file supposed to be here? :p

Reply:
Nope.